src/components/Testimonial/Testimonial.module.css

```diff
@@ -0,0 +1,79 @@
+.card {
+    flex: 1;
+    width: 100%;
+    margin: 1rem;
+    max-width: 32.625rem;
+    min-height: 11.75rem;
+}
+  
+.card__inner {
+    position: relative;
+    background-color: var(--white-40);
+    color: var(--primary-50);
+    border-radius: 1rem;
+    transition: all 0.2s ease-in-out;
+    
+    display: flex;
+    flex-direction: column-reverse;
+    align-items: flex-start;
+    padding: 1.5rem;
+    gap: 2rem;
+}
+
+@media (min-width: 798px) {
+    .card {
+        max-width: 65.250rem;
+    }
+    .card__inner {
+        flex-direction: row;
+        align-items: center;
+        padding: 2rem 4rem;
+        gap: 4rem;
+    }
+}
+
+.card__info {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    padding: 0px;
+    gap: 1rem;
+}
+
+.title {
+    font-weight: 800;
+    font-size: .75rem;
+    line-height: .875rem;
+    letter-spacing: 0.04em;
+    text-transform: uppercase;
+    color: var(--primary-500);
+}
+
+.content {
+    font-weight: 700;
+    width: 100%;
+    color: var(--primary-50);
+    background: var(--pink-gradient-200);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+}
+  
+.content span {
+    font-weight: 400;
+    background: var(--primary-50);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+}  
+
+.pink-gradient-200 {
+    color: var(--pink-gradient-200);
+}
+
+.source {
```

Comment:
there should be a typography that kinda matches this. at most you might only need to override the font weight i believe.

you can use this as a guide too

[dev hub typography and color styling](https://www.figma.com/file/bQARkk82bZF5XxoVft8EaF/BE-library?node-id=9282%3A27785)

if you don't have access, please ask @Zelig880